src/libopensc/card-tcos.c

```diff
@@ -436,9 +436,9 @@ static int tcos_list_files(sc_card_t *card, u8 *buf, size_t buflen)
 		r = sc_check_sw(card, apdu.sw1, apdu.sw2);
 		LOG_TEST_RET(ctx, r, "List Dir failed");
 		if (apdu.resplen > buflen) return SC_ERROR_BUFFER_TOO_SMALL;
-		sc_log(ctx, 
-			 "got %"SC_FORMAT_LEN_SIZE_T"u %s-FileIDs\n",
-			 apdu.resplen / 2, p1 == 1 ? "DF" : "EF");
+		sc_log(ctx,
+				"got %"SC_FORMAT_LEN_SIZE_T"u %s-FileIDs\n",
+				apdu.resplen / 2, p1 == 1 ? "DF" : "EF");
```

Comment:
+1 for removing the bogus spaces, but this seems to me as over-intentation. Similar problems are in the following diffs.